### DISCUSSION TITLE
The Cache Tag documentation not exists in Laravel 10 docs

### DISCUSSION THREAD

**Discussion by farshidpg** [laravel/framework#48947] [General]
The documentation of cache tag not exists in Laravel 10 cache documentation

theres no content for this link
https://laravel.com/docs/10.x/cache#cache-tags

But exists in older documentation version d for example 9.x or 8.x 

https://laravel.com/docs/9.x/cache#cache-tags

**Comment by shaedrich** [comment]
That's the commit in question: https://github.com/laravel/docs/commit/63dde39470588ab1ad39cea9a592f0838ca5ad47

There's some vivid discussion below, but as alway, Taylor doesn't bother to explain himself.

**Comment by crynobone** [comment]
See https://github.com/laravel/framework/pull/48078#issuecomment-1716332925
